cmd/podman/pods/create.go

```diff
@@ -291,3 +291,13 @@ func replacePod(name string) error {
 	}
 	return removePods([]string{name}, rmOptions, false)
 }
+
+func MapOptions() {
```

Comment:
Where is this called?

Reply:
Sorry, cherry picking from an old branch messed up this file, fixed now. It is called above `ToPodSpecGen` and combines the code that was there before into a function since more options are going to be added